src/SignalR/clients/ts/signalr/src/IConnection.ts

```diff
@@ -6,6 +6,7 @@ import { TransferFormat } from "./ITransport";
 /** @private */
 export interface IConnection {
     readonly features: any;
+    readonly connectionId?: string;
```

Comment:
Should this should be `string | null` since we are going to expose this?

Reply:
On HubConnection it should be.